pkg/ccl/streamingccl/streamproducer/batch_manager.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package streamproducer
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/kv/kvpb"
+	"github.com/cockroachdb/cockroach/pkg/repstream/streampb"
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+)
+
+type batchManager struct {
```

Comment:
When I see `manager` I typically think of something that oversees a collection of a thing. But, let's not bikeshed the name.

Reply:
refactored to `batcher`. we can further bikeshed after this pr merges :)